sections/4.1-error-handling.tex

```diff
@@ -94,15 +94,27 @@ \subsection{Promise 1: No null pointer dereferences}
     Ok(a + b)
 }
 \end{minted}
-\pause
+Note: Error signature must match!
+\end{frame}
+
+%%%
+
+\begin{frame}[fragile]{Ergonomic error handling with the \texttt{?} operator}
+\begin{minted}{rust}
+fn do_calc() -> Result<i32, String> {
+    let a = do_subcalc1()?;
+    let b = do_subcalc2()?;
```

Comment:
Why not just replace the `let b` call in the previous slide with a question mark, to show the two equivalent syntaxes?